docs/content/guides/author-apps/portable-resources/howto-author-portable-resources/index.md

```diff
@@ -0,0 +1,88 @@
+---
+type: docs
+title: "How-To: Author portable resources"
+linkTitle: "Author portable resources"
+description: "Learn how to author portable resources in Radius"
+weight: 200
+categories: "How-To"
+tags: ["portability"]
+---
+
+This guide will teach you how to author a portable resource for your [Radius Application]({{< ref "/guides/author-apps/application/overview" >}}).
+
+## Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius environment]({{< ref "installation#step-3-initialize-radius" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+
+## Step 1: Define a container resource
+
+Create a Bicep file `app.bicep`, add a container resource that will be leveraged by your portable resource later:
+
+{{< rad file="snippets/app.bicep" embed=true marker="//EMPTYCONTAINER" >}}
+
+## Step 2: Define your portable resource
+
+Portable resources provide **abstraction** and **portability** to Radius Applications. Radius currently offers options to manually provision the resources or automatically provision them via Recipes.
+
+{{< tabs Recipe Manual >}}
+
+{{% codetab %}}
+
+Recipes enable a separation of concerns between infrastructure operators and developers by automating infrastructure deployment. To learn more visit the [Recipes overview]({{< ref "/guides/recipes/overview" >}}). You can run a default recipe registered in your environment or select the specific Recipe you want to run. 
+
+#### Default Recipe
+
+{{< rad file="snippets/app-redis-recipe.bicep" embed=true marker="//Recipe" >}}
+
+#### Specifying a Recipe by name
```

Comment:
I think sub-choices within a tab choice is too confusing. Let's instead just have the code snippet be the default Recipe and we add a sentence that says "For more information on Recipes refer to the Recipe docs"